crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -419,3 +419,26 @@ x-----x
 x-----x
 xxb+bxx
 ENDMAP
+
+NAME:   dilly_ecumenical_altar_fadedgnrine
+TAGS:   no_monster_gen transparent
+MONS:   plant, gnoll; nothing
+SUBST: z = t111
+KITEM: E = never_decay gnoll shaman skeleton / never_decay gnoll skeleton
+FTILE: 12Et'_  = floor_grass
```

Comment:
```suggestion
FTILE: 1Et'_  = floor_grass
COLOUR: 1E'_ = green
```

Use the `COLOUR` directive to make sure console players also get a good experience.